index.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+//1. Вивід парних чисел циклом for
+
+let value = prompt();
+
+if (isNaN (value)) {
+    console.log ("Помилка: введіть число");
+} else { 
+    for (let i = value; i >= 1; i--) {
+        if (i != value && i % 2 === 0) {
```

Comment:
```suggestion
        if (i !== value && i % 2 === 0) {
```
Ніколи не використовуй нестроге порівняння, тільки строге. І загалом, в цьому рядку можна обійтись без перевірки i != value, нащо вона тут?